commands/choose.js

```diff
@@ -0,0 +1,16 @@
+module.exports.execute = async (client, message, args) => {
+	// Convert everything to string so the args can be parsed more easily
+	for (let i = 0; i < args.length - 1; i++) {
+		args[i] = args[i].toString();
+	}
```

Comment:
Is this needed? Which type does args have before converting them? (Hint, you can use [typeOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) to check this)